data_steward/cdr_cleaner/clean_cdr.py

```diff
@@ -157,6 +158,7 @@
     (remove_aian_participants.get_queries,),
     (validate_missing_participants.delete_records_for_non_matching_participants,
     ),
+    (DropDuplicatePpiQuestionsAndAnswers,),
```

Comment:
I noticed in the ticket it said this should be applied to the `RDR export`-- should this be moved up to `RDR_CLEANING_CLASSES`?

Reply:
Oh my bad! I'll change it. Thanks for finding it!